app/javascript/react/screens/App/Overview/components/InfrastructureMappingsList/helpers.js

```diff
@@ -47,15 +47,19 @@ export const mapInfrastructureMappings = (transformation_mapping_items, clusters
    * }
    *
    */
-  const clusterMappingItems = transformation_mapping_items.filter(
-    item => item.destination_type.toLowerCase() === 'emscluster'
-  );
-  const datastoreMappingItems = transformation_mapping_items.filter(
-    item => item.destination_type.toLowerCase() === 'storage'
-  );
-  const networkMappingItems = transformation_mapping_items.filter(
-    item => item.destination_type.toLowerCase() === 'lan'
-  );
+  const clusterMappingItems = transformation_mapping_items.filter(item => {
+    const destination = item.destination_type.toLowerCase();
+    return destination === 'emscluster' || destination === 'cloudtenant';
+  });
+  const datastoreMappingItems = transformation_mapping_items.filter(item => {
+    const destination = item.destination_type.toLowerCase();
+    return destination === 'storage' || destination === 'cloudvolume';
```

Comment:
Now that https://github.com/ManageIQ/manageiq-v2v/pull/673 is in, might as well update this now

Reply:
I'm on it